templates/Dockerfile-rpm.tmpl

```diff
@@ -75,6 +75,7 @@ RUN if [[ %%os%%   == oraclelinux ]] && [[ %%release%% == 7 ]]; then yum install
 
 # install build tools and PostgreSQL development files
 RUN ( yum install -y https://%%rpm_url%%  \
+    && ( [[ 16 != 16 ]] || sed -i '/\[pgdg16-updates-testing\]/{n;n;n;s/.*/enabled=1/}' /etc/yum.repos.d/pgdg-redhat-all.repo ) \
```

Comment:
I do not think that `[[ 16 != 16 ]]` part is correct. I guess you tried to do a comparison between 16 and a variable that stores the PG version in there.